routing_model/src/flows_dst.rs

```diff
@@ -347,6 +347,8 @@ impl CheckAndProcessElderChange {
     }
 }
 
+/// Implementing the flow diagram according to the state at
+/// https://raw.githack.com/maidsafe/routing/bd2d7d32cb1fcffc57e652df3fa5ff0f92f430fe/routing_model/index.html
```

Comment:
I think this is probably better at the PR level rather than in the code.
Just my preference though, but also it should apply to the full flow rather than the one bit.